Error: cannot export: SignatureSet

**Describe the bug**
I cannot compile nimbus_beacon_node 1.0.8

```
~/nimbus-eth2 tags/v1.0.8* user@gudja
❯ make -j $(nproc) nimbus_beacon_node
Building: build/nimbus_beacon_node
/home/user/nimbus-eth2/beacon_chain/spec/signatures_batch.nim(19, 8) Error: cannot export: SignatureSet
make: *** [Makefile:294: nimbus_beacon_node] Error 1
```


**To Reproduce**
Steps to reproduce the behavior:
1. Platform details (OS, architecture): Archlinux, kernel `Linux gudja 5.10.11-arch1-1 x86_64 GNU/Linux`
2. Branch/commit used: `tags/v1.0.8`
3. Commands being executed: `make -j $(nproc) nimbus_beacon_node`
4. Relevant log lines: `/home/user/nimbus-eth2/beacon_chain/spec/signatures_batch.nim(19, 8) Error: cannot export: SignatureSet`

**Additional context**
Compiling 1.0.6 that way used to work fine.

--- a/docs/the_nimbus_book/src/build.md
+++ b/docs/the_nimbus_book/src/build.md
@@ -27,3 +27,7 @@ To build the Nimbus beacon node and it's dependencies, run:
 make nimbus_beacon_node
 ```
 
+## Updating the node
+
+Make sure you stay on the lookout for any critical updates to Nimbus and [keep your node updated](./keep-updated.md).
+

